src/commercetools/testing/product_projections.py

```diff
@@ -75,38 +75,40 @@ def _convert_product_projection(
         self, product: types.Product, staged: bool = False
     ) -> typing.Optional[types.ProductProjection]:
         """Convert a Product object to a ProductProjection object"""
-        if product.master_data is None:
+        if product["masterData"] is None:
```

Comment:
Sadly we lose typing info here then?

Reply:
Yeah so an upcoming PR will introduce a custom predicate parser which allows us to mock query methods. For this to work we need to store raw data instead of typed objects in the mocked objects :-)